.circleci/config.yml

```diff
@@ -12,12 +12,28 @@ orbs:
   gcp-gcr: circleci/gcp-gcr@0.0.4
 
 jobs:
+  resolve_latest:
+    executor: docker-node
+    steps:
+      - checkout
+      - run:
+          name: Resolve latest tbtc.js
+          command: |
+            npm update @keep-network/tbtc.js
+            echo "Resolved tbtc.js version " $(jq '.dependencies["@keep-network/tbtc.js"]')
```

Comment:
Where is `jq` getting its JSON from here?

Reply:
I actually don't need to do this, because `npm update` shows the version when it does a bump.

Reply:
Fixed.